docs/tutorial.md

```diff
@@ -0,0 +1,380 @@
+pipenv, version 2020.11.15
+...
+```
+
+### Writing the tests
+
+We'll now write some basic tests for the HelloWorld app we've built above in
+[building your app](#building-your-app).
+
+#### Preparing environment
+
+Let's create a directory for storing tests. `abs` looks for them in the `tests/abs`
+directory, so let's start a fresh python virtual env there:
+
+```bash
+$ mkdir -p examples/tutorial/tests/abs
+$ cd examples/tutorial/tests/abs
+$ pipenv --python 3.8
+Creating a virtualenv for this project...
+Pipfile: /home/piontec/work/giantswarm/git/app-build-suite/examples/tutorial/tests/abs/Pipfile
+Using /home/piontec/.virtualenvs/app-build-suite-pOZwqM8M/bin/python3.8 (3.8.6) to create virtualenv...
+⠦ Creating virtual environment...created virtual environment CPython3.8.6.final.0-64 in 362ms
+  creator CPython3Posix(dest=/home/piontec/.virtualenvs/abs-4WvsKJg-, clear=False, no_vcs_ignore=False, global=False)
+  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/piontec/.local/share/virtualenv)
+    added seed packages: pip==21.0.1, setuptools==56.0.0, wheel==0.36.2
+  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator
+
+✔ Successfully created virtual environment!
+Virtualenv location: /home/piontec/.virtualenvs/abs-4WvsKJg-
+Creating a Pipfile for this project...
+```
+
+Now, we need to add our dependencies. If we're going to use `pytest-helm-chart`,
+everything else will come as dependencies:
+
+```bash
+$ pipenv install "pytest-helm-charts>=0.3.1"
+Installing pytest-helm-charts>=0.3.1...
+Adding pytest-helm-charts to Pipfile's [packages]...
+✔ Installation Succeeded
+Pipfile.lock not found, creating...
+Locking [dev-packages] dependencies...
+Locking [packages] dependencies...
+Building requirements...
+Resolving dependencies...
+✔ Success!
+Updated Pipfile.lock (a62443)!
+Installing dependencies from Pipfile.lock (a62443)...
+  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 0/0 — 00:00:00
+```
+
+#### Implementing tests
+
+Now we can start implementing actual tests. To get a full sample source code,
+just copy the `test_example.py` [file](../examples/apps/hello-world-app/tests/abs/test_example.py) to our `tests/abs` directory, so it looks like this:
+
+```bash
+$ ls
+Pipfile  Pipfile.lock  test_example.py
+```
+
+The simplest test case code in the `test_example.py` file looks like this:
+
+```python
+from typing import cast
+
+import pytest
+import pykube
+from pytest_helm_charts.fixtures import Cluster
+
+
+@pytest.mark.smoke
+def test_we_have_environment(kube_cluster: Cluster) -> None:
+    assert kube_cluster.kube_client is not None
+    assert len(pykube.Node.objects(kube_cluster.kube_client)) >= 1
+```
+
+In this test, we're only checking if we can get a working connection object to
+work with our cluster. This is done by requesting the `kube_cluster: Cluster` object
```

Comment:
Do we have a list of available fixtures? 

Also do I always have `kube_cluster: Cluster`? I am using `kubernetes_cluster` - is this something different?
https://github.com/giantswarm/kyverno-policies/blob/1e2540f3d4f87da05f555e84c258cb9bd0d2cc03/helm/policies-common/tests/abs/test_basic_cluster_default.py#L17

Reply:
Yes, it's something different - I don't even know where is that injected from. But in general, fixtures are bound using their names, so something else is providing this fixture you're using. 
The list of available fixtures is here: https://github.com/giantswarm/pytest-helm-charts/blob/master/pytest_helm_charts/fixtures.py